src/solver.cpp

```diff
@@ -1881,10 +1973,6 @@ lbool Solver::execute_inprocess_strategy(
             if (conf.do_distill_clauses) {
                 distill_long_cls->distill(false);
             }
-        } else if (token == "str-impl") {
-            if (conf.doStrSubImplicit) {
-                dist_impl_with_impl->str_impl_w_impl();
```

Comment:
Why is this removed? This will never get pulled into mainline CryptoMiniSat with this removed.